config/environments/production.rb

```diff
@@ -134,4 +134,19 @@
 
   # Add the GraphQL probe for Skylight.
   config.skylight.probes << "graphql"
+
+  # Add throttling to application
+  rules = [
+    { method: "POST", limit: 10 },
+    { method: "POST", path: "/graphql", limit: 10 }]
+
+  require 'dalli'
```

Comment:
> The issue that we discussed in the last review call, was to change the Rails application cache store from dalli_store to mem_cache_store which is changed above in the same file.

@mahesh-krishnakumar That's not exactly how I remember it because... what does this change achieve? Given that we apparently cannot switch to using `mem_cache_store` _alone_, we're using that in _addition_ to `dalli`?

Why use two clients where we could use just one?

Reply:
@harigopal : Have upgraded `dalli` to mitigate this issue: https://blog.heroku.com/cache-invalidation-rails-5-2-dalli-store and switched to using `dalli_store` for the Rails cache, thus sticking to using one client.
0c3fa6b6